### PR TITLE
co-processor: add messages

### PR DESCRIPTION
**Pull request by disksing**
@pingcap/eng PTAL

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM @coocood @shenli PTAL

File: proto/coprocessor.proto [hunk 1]
```diff
@@ -0,0 +1,36 @@
+syntax = "proto2";
+package coprocessor;
+
+import "errorpb.proto";
+import "kvrpcpb.proto";
+
+message Request {
+    optional kvrpcpb.Context context   = 1;
+    optional int64           tp        = 2;
```
Comment (ngaut) [inline proto/coprocessor.proto]: Please add todo, change to  enum later.

**Comment by shenli** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Comment by disksing** [comment]
Updates: add txn version in request (need by MVCC)
PTAL @ngaut @shenli @coocood

**Comment by coocood** [comment]
@disksing txn version is included in `SelectRequest.start_ts` already.

**Comment by disksing** [comment]
@coocood Thanks. I will revert the additional commit.

**Comment by coocood** [comment]
LGTM

File: proto/coprocessor.proto (hunk 1 above)
Comment (coocood) [inline proto/coprocessor.proto]: I added an issue.

**Pull request merged**